packages/common-test-utils/src/index.ts

```diff
@@ -810,3 +810,104 @@ export class NoteTestPresetsV2 {
     ];
   }
 }
+
+/**
+ * Utility class to make the tests against vault to be concise while having descriptive
+ * failure messages. */
+export class VaultAsserter {
+  private readonly workspaceVault: WorkspaceVault;
```

Comment:
`WorkspaceVault` is a [vault that belongs to another workspace](https://wiki.dendron.so/notes/6682fca0-65ed-402c-8634-94cd51463cc4.html#vault-types).

Reply:
Will rename to just vault.

Reply:
removed